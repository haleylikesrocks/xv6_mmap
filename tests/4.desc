Stress test : Testing Anonymous mmap that spans across multiple pages with PROT_WRITE flag
