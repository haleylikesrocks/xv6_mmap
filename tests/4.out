XV6_TEST_OUTPUT : mmap good
XV6_TEST_OUTPUT : strlen(r) = 12287
XV6_TEST_OUTPUT : Return val of strcmp 0
XV6_TEST_OUTPUT : value at the newly mapped anonymous memory region is correct.!!
XV6_TEST_OUTPUT : munmap good
